packages/shared/src/components/Button.jsx

```diff
@@ -1,5 +1,50 @@
 import React from 'react';
+import Button from '@mui/material/Button';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
 
-export default function Button() {
-  return <button type="submit">TEST BUTTON</button>;
+function ButtonLogin({
+  type,
+  color,
```

Comment:
Try to rename the names of the component equal to the filename in UpperCamelCase, and other best practice is put a folder with the name of the main component, and a file index inside the component.